pkg/runtime/cli.go

```diff
@@ -120,10 +122,18 @@ func FromFlags() (*DaprRuntime, error) {
 	var globalConfig *global_config.Configuration
 	var configErr error
 
+	var certChain *certchain.CertChain
```

Comment:
may be i am missing something here, but this variable certChain is not assigned any value, wouldn't it be nil always when passed to GetOperatorClient below(line 136) which would return a client without tls?
Should it pass runtimeConfig.CertChain instead as thats properly assigned.

Reply:
When mTLS is disabled, it will return a non secured connection.

Reply:
but this var "certChain" is not being set to anything when mTLS is enabled so GetOperatorClient  will always return a client without tls when this "certChain" var is passed to it.

Reply:
oh good catch. fixed now.